addons/l10n_at/data/template/account.account-at.csv

```diff
@@ -132,66 +149,82 @@
 "chart_at_template_3542","Verrechnung Dienstgeberzuschlag","3542","True","liability_current","l10n_at.account_tag_l10n_at_PCVIII1"
 "chart_at_template_3550","Verrechnung Kommunalsteuer","3550","False","liability_current","l10n_at.account_tag_l10n_at_PCVIII1"
 "chart_at_template_3551","Verrechnung Wiener Dienstgeberabgabe","3551","False","liability_current","l10n_at.account_tag_l10n_at_PCVIII1"
-"chart_at_template_3600","Verrechungskonto Sozialversicherung","3600","True","liability_payable","l10n_at.account_tag_l10n_at_PCVIII2"
-"chart_at_template_3610","Verrechnungskonto Magistrat/Gemeinde (KoSt, U-Bahn, etc.)","3610","True","liability_payable","l10n_at.account_tag_l10n_at_PCVIII2"
+"chart_at_template_3600","Verbindlichkeiten im Rahmen der sozialen Sicherheit","3600","True","liability_payable","l10n_at.account_tag_l10n_at_PCVIII2"
```

Comment:
No go, these accounts are too important to be replaced in this way...please revert this...

Reply:
What is the purpose of these accounts and can you provide a mapping to the SAF-T CoA?

Reply:
it is basically a payable account which is keeping track of open positions in the month they become due. You can map both to 3600 in SAF-T which obviously is available there...